src/commands/apm_cmds/version.js

```diff
@@ -1,7 +1,31 @@
-const fs = require('fs')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const fs = require('fs-extra')
```

Comment:
Switched to fs-extra as it's basically fs with flowers. Plus it has been around already anyways.